sycl/doc/extensions/SPIRV/SPV_INTEL_function_pointers.asciidoc

```diff
@@ -178,6 +188,18 @@ but it's address can be obtained and passed into an Entry Point for further
 usage via *OpFunctionPointerCallINTEL*. This function must not be optimized
 out based on call graph/reachability analysis 2+||
 *IndirectReferencesINTEL*|
+6409 | *ArgumentAttributeINTEL* +
+Apply to an *OpFunctionPointerCallINTEL* to preserve _Function Parameter Attribute_ of it's
+argument with the specified _Index_. +
+_Function Parameter Attribute_ and _Index_ are literal strings. _Optional Argument_ is literal string to preserve alignment value, it can exist if the _Function Parameter Attribute_ is an alignment
```

Comment:
Why strings? I think having them as integers is much more convenient. AFAIK there is no alignment attribute in https://www.khronos.org/registry/SPIR-V/specs/unified1/SPIRV.html#Function_Parameter_Attribute, why do we need extra optional argument?
```suggestion
_Index_ is an unsigned 32-bit integer indicating the argument number _N_. The attribute is applied to _Argument N_ of *OpFunctionPointerCallINTEL*. _Argument Attribute_ specifies which attribute is applied, its possible values are enumerated in section "Function Parameter Attribute" of the core SPIR-V specification.
```

Reply:
If we choose to keep only the attributes contained in the enumeration, this is not necessary. Otherwise, it's necessary. What do you think?